File: src/test/console/java/src/main/java/util/MyTouchableWebDriver.java
package util;

import org.openqa.selenium.interactions.HasTouchScreen;
import org.openqa.selenium.interactions.TouchScreen;
import org.openqa.selenium.remote.DesiredCapabilities;
import org.openqa.selenium.remote.RemoteTouchScreen;
import org.openqa.selenium.remote.RemoteWebDriver;
import org.openqa.selenium.remote.HttpCommandExecutor;

import java.net.URL;

public class MyTouchableWebDriver extends RemoteWebDriver
implements HasTouchScreen {
  private TouchScreen touch;
  public MyTouchableWebDriver(URL remoteAddress, DesiredCapabilities desiredCapabilities) {
    super(remoteAddress, desiredCapabilities );
    touch = new RemoteTouchScreen(getExecuteMethod());
  }

  public MyTouchableWebDriver(HttpCommandExecutor executor, DesiredCapabilities desiredCapabilities) {
    super(executor, desiredCapabilities);
    touch = new RemoteTouchScreen(getExecuteMethod());
  }

  public TouchScreen getTouch() {
    return touch;
  }
}

File: src/test/console/dotnet/Jsonwired/util/MyTouchableWebdriver.cs
using System;
using OpenQA.Selenium;
using OpenQA.Selenium.Remote;

namespace util {
	public class MyTouchableWebDriver : RemoteWebDriver, IHasTouchScreen {
		public ITouchScreen TouchScreen { get; }

		public MyTouchableWebDriver(Uri remoteAddress, DesiredCapabilities desiredCapabilities) : 
		base(remoteAddress, desiredCapabilities, TimeSpan.FromSeconds(240)) {
			TouchScreen = new RemoteTouchScreen(this);
		}
  }
}